fdbserver/workloads/MachineAttrition.actor.cpp

```diff
@@ -35,6 +35,21 @@ static std::set<int> const& normalAttritionErrors() {
 	return s;
 }
 
+ACTOR Future<Void> resetHealthyZoneAfter(Database cx, double duration) {
+	state Transaction tr(cx);
+	loop {
+		try {
+			tr.setOption(FDBTransactionOptions::LOCK_AWARE);
+			wait(delay(duration));
```

Comment:
make the delay state Future<Void> delayFuture = delay(duration); outside the loop